content/scout/dashboard.md

```diff
@@ -111,11 +111,60 @@ more information on how to get started with integrations, see
 
 The settings menu in the Docker Scout Dashboard contains:
 
-- **Billing** for managing your Docker Scout subscription and payments
-- **Repository settings** for enabling and disabling repositories
+- [**Billing**](#billing-settings) for managing your Docker Scout subscription and payments
+- [**Repository settings**](#repository-settings) for enabling and disabling repositories
+- [**Notifications](#notification-settings) for managing your notification preferences for Docker Scout.
+
+### Billing settings
+
+The [Billing settings](https://scout.docker.com/settings/billing) page shows
+you the Docker Scout plan for the current organization. Here you can see the
+what's included in your plan, compare it with other available plans, and change
+the plan if you're an organization owner.
+
+For more information about subscription plans, see
+[Docker Scout subscriptions and features](../subscription/scout-details.md)
+
+### Repository settings
 
 When you enable Docker Scout for a repository,
 Docker Scout analyzes new tags automatically when you push to that repository.
 To enable repositories in Amazon ECR, Azure ACR, or other third-party registries,
 you first need to integrate them.
 See [Container registry integrations](./integrations/_index.md#container-registries)
+
+### Notification settings
+
+The [Notification settings](https://scout.docker.com/settings/notifications)
+page is where you can change the preferences for receiving notifications from
+Docker Scout. Notification settings are personal, and preferences apply to all
+organizations that you're a member of. Changing notification settings only
+affects your personal account, not the entire organization.
+
+The purpose of notifications in Docker Scout is to raise awareness about
+upstream changes that affect you. Docker Scout will notify you about when a new
+vulnerability is disclosed in a security advisory, and it affects one or more
+of your images. You will not receive notifications about changes to
+vulnerability exposure or policy compliance as a result of pushing a new image.
+
+> **Note**
+>
+> Notifications are only triggered for the *last pushed* image tags for each
+> repository. "Last pushed" refers to the image tag that was most recently
+> pushed to the registry and analyzed by Docker Scout. If the last pushed image
+> is not by a newly disclosed CVE, then no notification will be triggered.
+
+The available notification settings are:
+
+- **In-app notifications**: select this check-box to receive notification in
```

Comment:
**discussion:** I don't think our naming scheme for this is solid but in-app notifications means something **slightly** different. Notifications **always** land on the notification center, regardless of settings.

- **When both in-app and OS are off**, all notifications come "silently" to the notification center. You'll see the count of notifications go up in next to the bell icon. In this situation you won't get a snackbar/toast, sound nor an OS-level notification
- **In-app notifications** solely controls whether or not we **alert** you, by showing a toast/snackbar, when a notification comes it
- **OS notifications** - this is on point

Reply:
OK, so should we consider changing the label for this option? Maybe something like "Notification pop-ups"?

Reply:
I'm thinking we should probably do it, yes 🤔 Do you think "notification pop-ups" is the way to go?

Reply:
Let's go ahead with notification pop-ups 🙏 We're going to change this on the frontend

Reply:
This is done!

Reply:
Updated the doc 👍🏻